extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -159,6 +202,12 @@ export class SystemLayer {
 
     /** @const @private {!../../../src/service/vsync-impl.Vsync} */
     this.vsync_ = Services.vsyncFor(this.win_);
+
+    /** @const @private {!../../../src/service/timer-impl.Timer} */
+    this.timer_ = Services.timerFor(this.win_);
+
+    /** @private {?(number|string)} */
+    this.muteMessageTimeout_ = null;
```

Comment:
Is this used?

Reply:
Oops meant to delete it! thanks for all the feedback btw!